template.go

```diff
@@ -210,60 +208,12 @@ func funcMap(i *funcMapInput) template.FuncMap {
 		"caRoots":      connectCARootsFunc(i.store, i.used, i.missing),
 		"caLeaf":       connectLeafFunc(i.store, i.used, i.missing),
 
-		// scratch
-		"scratch": func() *scratch { return &scrat },
-
-		// Helper functions
-		"base64Decode":    base64Decode,
-		"base64Encode":    base64Encode,
-		"base64URLDecode": base64URLDecode,
-		"base64URLEncode": base64URLEncode,
-		"byKey":           byKey,
-		"byTag":           byTag,
-		"contains":        contains,
-		"containsAll":     containsSomeFunc(true, true),
-		"containsAny":     containsSomeFunc(false, false),
-		"containsNone":    containsSomeFunc(true, false),
-		"containsNotAll":  containsSomeFunc(false, true),
-		"env":             envFunc(i.env),
-		"executeTemplate": executeTemplateFunc(i.t),
-		"explode":         explode,
-		"explodeMap":      explodeMap,
-		"in":              in,
-		"indent":          indent,
-		"loop":            loop,
-		"join":            join,
-		"trimSpace":       trimSpace,
-		"parseBool":       parseBool,
-		"parseFloat":      parseFloat,
-		"parseInt":        parseInt,
-		"parseJSON":       parseJSON,
-		"parseUint":       parseUint,
-		"parseYAML":       parseYAML,
-		"plugin":          plugin,
-		"regexReplaceAll": regexReplaceAll,
-		"regexMatch":      regexMatch,
-		"replaceAll":      replaceAll,
-		"sha256Hex":       sha256Hex,
-		"timestamp":       timestamp,
-		"toLower":         toLower,
-		"toJSON":          toJSON,
-		"toJSONPretty":    toJSONPretty,
-		"toTitle":         toTitle,
-		"toTOML":          toTOML,
-		"toUpper":         toUpper,
-		"toYAML":          toYAML,
-		"split":           split,
-		"byMeta":          byMeta,
-		"sockaddr":        sockaddr,
-		// Math functions
-		"add":      add,
-		"subtract": subtract,
-		"multiply": multiply,
-		"divide":   divide,
-		"modulo":   modulo,
-		"minimum":  minimum,
-		"maximum":  maximum,
+		// ???
```

Comment:
> ???

Very interested in its final form

Reply:
Right. Got thinking about the library idea and forgot that stuff was still in there. I'll strip it out.

Reply:
A bit more to it with tests and trying to move things into tfunc/ instead of just deleting them. I'll have this done in a bit.

Reply:
These are not stripped out. It should be merge-ready now.

Reply:
The latest changes I can see only include deletes -- are you still planning to move them to `tfunc`? if so might be missing those files 😄

Reply:
not/now ... "now stripped out"